__tests__/label.test.js

```diff
@@ -1,22 +1,23 @@
-import { random } from 'faker';
 import {
-  clearDatabaseState, getAuthenticatedUser, launchApp, shutdownApp,
+  create, database, getAuthenticatedUser, launchApp, shutdownApp,
 } from './helpers';
 
 describe('Label', () => {
   let app;
   let cookies;
+  let db;
 
   beforeAll(async () => {
     app = await launchApp();
+    db = database(app);
   });
 
   afterAll(async () => {
     await shutdownApp(app);
   });
 
   beforeEach(async () => {
-    await clearDatabaseState(app);
+    await db.clear();
```

Comment:
state should be cleaned after the test passed, not before. otherwise we will have not clean state after the last test

Reply:
done